### PR TITLE
Bathymetry from COG-Server, Dataset Management from COG-Maker

### PR DESCRIPTION
**Pull request by lowriech**

### CODE REVIEW

File: CogServer/app.py [hunk 1]
```diff
@@ -25,8 +27,26 @@
...
+    
+    # Data Prep
+    data = (data + offset) / scale
+    data = data * nodata_to_zero(data, nodata)
+    
+    # RGB png band creation
+    r = np.floor(data / math.pow(256,2)) % 256
+    g = np.floor(data / 256) % 256
+    b = np.floor(data) % 256
+    return np.array([r,g,b]).astype('uint8')
```
Comment (gislawill) [inline CogServer/app.py:47]: Very cool

File: CogServer/app.py [hunk 2]
```diff
@@ -53,17 +74,40 @@ def tile(
...
+    unscale: bool = True
+    ):
+    '''Returns a Mapbox-ready elevation tile'''
+    options = img_profiles.get('png')
+    dataset = f'{GCS_BASE}/{dataset}'
+    options={"unscale":unscale}
+    with Reader(dataset, options=options) as cog:
+        t = cog.tile(x, y, z)
+        buff = render(
+            to_rgb(t.data[0], cog.info().nodata_value),
```
Comment (gislawill) [inline CogServer/app.py:102]: Is this pretty fast? Seems like a good opportunity for preprocessing (and saving new files) if there are any performance concerns

**Review by gislawill — APPROVED** [review]

File: CogServer/app.py (hunk 2 above)
Comment (lowriech) [reply CogServer/app.py:102]: It's fast <3

**Comment by lowriech** [comment]
I added some updates here, with the intention that this will be the first foray into managed metadata.  I'll keep putting this together over the next week for a more fully fleshed out POC.

**Pull request merged**